doc/themes/kconfig/versions.html

```diff
@@ -0,0 +1,21 @@
+{# Add rst-badge after rst-versions for small badge style. #}
+  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
+    <span class="rst-current-version" data-toggle="rst-current-version">
+      <span class="fa fa-book">Kconfig reference</span>
```

Comment:
```suggestion
      <span class="fa fa-book"> Kconfig reference</span>
```
The main point was actually the added space - it will look strange otherwise.

Reply:
Fixed